ui/util/url.js

```diff
@@ -0,0 +1,68 @@
+const LBRY_INC_DOMAINS = ['lbry.io', 'lbry.com', 'lbry.tv', 'lbry.tech', 'lbry.fund', 'spee.ch'];
```

Comment:
Make this an ENV variable (or configurable/overrideable in some way, though it's good if this is the default without explicit configuration).

Call it `WEB_DOMAINS`

Reply:
This isn't just web domains. We use it for triggering a warning when opening links that _aren't_ from one of these domains. Maybe `TRUSTED_DOMAINS`?

Reply:
I fixed this problem in #3312 :grin: